### PR TITLE
[WIP]New chaplain holy weapon golden monkey idol

### PR DESCRIPTION
**Pull request by improvedname**
Adds a chaplain holy weapon, Golden monkey idol
The monkey idol can't be fit in your backpack.
The monkey idol grants you the jungle faction which let wild gorilla's not attack you
The monkey idol has terrible melee damage but makes up for being a heavy throwing weapon

Monkey idol looks like (current):
image

Vs old:
image

:cl: Improvedname
add: Adds the golden monkey idol
🆑 /

- [x]  Better sprite

- [ ] Code

### CODE REVIEW

**Comment by improvedname** [comment]
gonna rework dis

**Comment by improvedname** [comment]
Planning to rework this to give the jungle faction to whenever your holding it and dropping it nakes you lose it

File: modular_citadel/code/game/objects/items/melee/holy_weapons.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
+obj/item/nullrod/monkeyidol
+	name = "Golden monkey idol"
+	desc = "Great authority dwells within this idol"
+	icon_state = "Monkeyidol"
+	item_state = "Monkeyidol"
+	icon = 'modular_citadel/icons/obj/holyweapons.dmi'
+	force = 5
+	throwforce = 20
```
Comment (Poojawa) [inline modular_citadel/code/game/objects/items/melee/holy_weapons.dm:8]: > 20 throwforce

**Review by Poojawa — CHANGES_REQUESTED** [review]
Null rods only have 10.

File: modular_citadel/code/game/objects/items/melee/holy_weapons.dm (hunk 1 above)
Comment (GuppyLaxx) [reply modular_citadel/code/game/objects/items/melee/holy_weapons.dm:8]: @Poojawa >never heard of the atheists fedora

**Pull request closed without merging**